Dockerfile

```diff
@@ -1,9 +1,15 @@
-FROM debian:12 as builder
+FROM golang:1.21.3-bullseye as builder
+
 ARG TARGETARCH
```

Comment:
This line can be removed.

Reply:
Done https://github.com/open-telemetry/opentelemetry-go-instrumentation/pull/395/commits/2b924878f9fe89f58f46c63f2ee87b7863a0d14f